helm/oncall/templates/telegram-polling/deployment.yaml

```
@@ -0,0 +1,55 @@
+{{- if .Values.telegramPolling.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "oncall.telegramPolling.fullname" . }}
+  labels:
+    {{- include "oncall.telegramPolling.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "oncall.telegramPolling.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "oncall.telegramPolling.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "oncall.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
+        {{- if eq .Values.database.type "mysql" }}
+        {{- include "oncall.mariadb.wait-for-db" . | indent 8 }}
+        {{- end }}
+        {{- if eq .Values.database.type "postgresql" }}
+        {{- include "oncall.postgresql.wait-for-db" . | indent 8 }}
+        {{- end }}
+      containers:
+        - name: telegram-polling
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ['sh', '-c', 'python manage.py start_telegram_polling']
+          env:
+            {{- include "snippet.oncall.env" . | nindent 12 }}
+            {{- include "snippet.oncall.telegram.env" . | nindent 12 }}
+            {{- if eq .Values.database.type "mysql" }}
+            {{- include "snippet.mysql.env" . | nindent 12 }}
+            {{- end }}
+            {{- if eq .Values.database.type "postgresql" }}
+            {{- include "snippet.postgresql.env" . | nindent 12 }}
+            {{- end }}
```

similar to below comment, [new snippet](https://github.com/grafana/oncall/pull/2751/files#diff-e3f09fcc93f2a991e715f7c7908075e5ffc64d4ed1ac806c389f279b43bfb596R534-R544): 
```suggestion
            {{- include "snippet.db.env" . | nindent 12 }}
```